lib/helpers/buildURL.js

```diff
@@ -51,7 +51,7 @@ module.exports = function buildURL(url, params, paramsSerializer) {
         } else if (utils.isObject(v)) {
           v = JSON.stringify(v);
         }
-        parts.push(encode(key) + '=' + encode(v));
+        parts.push(encodeKey(key) + '=' + encodeURIComponent(v));
```

Comment:
should we just use `encodeURIComponent` for both key and value? what happens if we do that?